api/sessions_api.go

```diff
@@ -386,60 +448,3 @@ func (s *Session) handleAbnormalClosure() int {
 		return ConnLoopCodeContinue
 	}
 }
-
-// // Restarting the game means we play the game with
```

Comment:
Can we remove this commented code?

Reply:
I really wonder why this happens. I don't have this code on my local repo!!!

Reply:
probably some stale changes, rebasing with main before pushing could solve it.